### DISCUSSION TITLE
Migrating Personal to Enterprise, don't receive admin email to login

### DISCUSSION THREAD

**Discussion by robinkwilson** [mozilla/hubs#3025] [General Questions & Discussion] [answered]
@MegaMotion

The backstory: I started a Personal stack on my AWS account, then realized that it did not allow multiple servers, which I potentially want to have, so I deleted this stack and added a new Enterprise Multi-Server stack. Everything went smashingly until I tried to sign in to my new stack, but I waited and waited and the email never came, when previously it only took a few seconds. I don't recall my exact timing but I may have started creating the enterprise stack before the personal stack had finished deleting, and they are using the same URL and credentials - is there any chance the first one failed to let go of a resource by the time the second one needed to have it, or something?
Thanks!

**Comment by robinkwilson** [comment, marked as answer]
It's certainly possible you had collisions with dependencies on the Personal deleting stack + Enterprise spinning up stack is possible. AFAIK, the resources aren't locked down while one is using it or the other is adding to it. 

In a perfect world, I'd suggest you delete the Personal stack first. Check your domain's Hosted Zones for left over CNAMES + MX records from the old Personal Stack, delete those mentioned in [Solution 4 here](https://hubs.mozilla.com/docs/hubs-cloud-aws-troubleshooting.html#potential-solutions). Then spin up the Enterprise stack.

**Quick checks for you:**
1. Check that your admin email address is verified with amazon: https://hubs.mozilla.com/docs/hubs-cloud-aws-troubleshooting.html#in-my-hubs-admin-panel-i-see-networkerror-or-not-found-page-or-no-data-populates-in-any-of-the-admin-menus

2. In aws console SES, can you double check that an email is being sent when you try to login?

3. Add email addresses manually (Solution 1) so SES will send the email to them: https://hubs.mozilla.com/docs/hubs-cloud-aws-troubleshooting.html#solution-options

**Reply by MegaMotion** [reply]
Thanks Robin! And thanks for the reminder about the whole sandbox thing, I had forgotten about that. Don't need to worry about it today but I might soon.

Anyway, my email is there, and I can send a test email to myself. I can see that five emails have been sent, but that is probably from the original Personal server. Does it make sense at this point to wipe and start over?

**Reply by MegaMotion** [reply]
Oh also, re: those CNAMEs, in the guide you linked it says this: "Delete the CNAMEs from left from the last Hubs stack deployment -- Careful not to delete any CNAMEs that your domains use for another service". I am doing nothing with this URL except this hubs stack, does that mean I can just delete all the CNAMEs I see? Or are there particular ones I need to delete and others I need to leave?

**Reply by MegaMotion** [reply]
Answering my own question, hoping I got it right: there is one CNAME attached to each of my URLs which looks to be connected to AWS' acm-validation service, so I left that, but deleted everything else including all the temporary URLs with cute instance names. Crossing fingers...

**Reply by MegaMotion** [reply]
AHA, hahaha, whoops sorry! Newbie blunder, I forgot to check my spam folder. *sorry* :-p

**Reply by robinkwilson** [reply]
Yay for spam folders!

**Comment by MegaMotion** [comment]
Last question though: is it normal that my mail is coming from the US East - Virginia region, while my stack is installed in US West - Oregon?

**Reply by robinkwilson** [reply]
Correct. ALL emails will originate from N. Virginia (us-east-1) regardless of deploy region. 

You'll ask for an aws SES limit increase in N. Virginia via the docs: https://hubs.mozilla.com/docs/hubs-cloud-aws-troubleshooting.html#solution-options
